pages/interface/components/PasswordInput/index.js

```diff
@@ -0,0 +1,81 @@
+import { useState } from 'react';
+import { FormControl, IconButton, TextInput, Tooltip } from '@primer/react';
+import { EyeClosedIcon, EyeIcon } from '@primer/octicons-react';
+
+export default function PasswordInput(props) {
+  const { inputRef, id, name, label, errorObject, setErrorObject } = props;
+
+  const [isPasswordVisible, setIsPasswordVisible] = useState(false);
+  const [capsLockWarningMessage, setCapsLockWarningMessage] = useState(false);
+
+  function focusAfterEnd(ref) {
+    setTimeout(() => {
```

Comment:
Precisa mesmo desse `setTimeout`?

Reply:
Precisa, porque o comportamento padrão do `primer` é ser focar no início do texto. Eu não consegui remover esse comportamento. Então a solução que encontrei foi colocar esse `SetTimeout` só para ter um `delay` e focar no final do texto, mas é imperceptível para o usuário.

Reply:
A parte `ref.current.focus();` essa, sim, da para remover. Porque o botão já estará focado, mas eu quis deixar mais pelo fato de ficar explicito mesmo.